src/less/resources.less

```diff
@@ -245,6 +215,51 @@
       opacity: @global-opacity-disabled;
     }
   }
+
+  // We need to be a bit more relaxed with our borders since the container query watcher can get mixed in next to each `<Resource />` as state changes
```

Comment:
Spacing looks like it's off here for the comment and body of this rule.

Reply:
Could I see a screen shot of what you're seeing? Is it looking different than in the app currently? I did need to change the wrapper, so definitely good to catch anything that looks off. This specific item is about "when" to show the border.

Reply:
Sorry, ignore this comment. I thought the tilde character was something in Githubs diff instead of the CSS selector.